webapp/style/main.scss

```diff
@@ -96,5 +96,33 @@ body {
   overflow: hidden;
 }
 
+// buttons
+.btn-primary, .btn-cancel {
+  font-weight: 600;
+  min-width: 80px;
+}
+
+.btn-primary {
+  background-color: $blue;
+  color: $white;
+  text-transform: uppercase;
+
+  &:hover {
+    background-color: lighten($blue, 15%);
+  }
+}
 
+.btn-cancel {
+  color: $blue;
+}
 
+.button-bar {
+  display: flex;
+  align-self: center;
+  align-items: center;
+  justify-content: center;
+
+  button {
+    margin: 0 .5rem;
+  }
+}
```

Comment:
I would add these styles where the other button styles are defined (formBase.scss)

Reply:
ok